src/Workload/Microsoft.Maui.Sdk/Sdk/Microsoft.Maui.Sdk.Before.targets

```diff
@@ -21,4 +21,14 @@
     <_MauiUsingDefaultRuntimeIdentifier>true</_MauiUsingDefaultRuntimeIdentifier>
   </PropertyGroup>
 
+  <!-- Feature flags -->
+  <PropertyGroup>
+    <XamlLoadingIsEnabled Condition="'$(XamlLoadingIsEnabled)' == '' and '$(PublishAot)' == 'true'">false</XamlLoadingIsEnabled>
```

Comment:
nit: naming
Would it make sense to name the MSBuild property as: `EnableXamlLoading` while the feature flag can stay as `IsXamlLoadingEnabled`?

Reply:
+1 on `EnableXamlLoading` and we usually use the same name in C# code as well to avoid confusion.

Reply:
Now that I'm looking at the runtime feature switches, I see we often use `Support` support suffix for MSBuild properties and then the C# property is in the form of `Is<Feature>Supported` (for example `DynamicCodeSupport ->  IsDynamicCodeSupported`). What would you say to `XamlLoadingSupport -> IsXamlLoadingSupported`? Or do you prefer `EnableXamlLoading`?

Reply:
There was a discussion on this topic some time ago... but I just can't find it now. That said I vaguely remember that the pattern you deduced was the "recommendation".

Reply:
I think I still prefer `EnableXamlLoading` but I don't feel too strongly.

Reply:
@akoeplinger Do you think it would be confusing? I can see a few MSBuild properties that have the `Maui` prefix (`MauiUseDefaultAotProfile`, `MauiUseLocalPacks`, ...). Avalonia has for example `AvaloniaUseCompiledBindingsByDefault`. So maybe the feature flag pattern could be `MauiEnableXamlLoading` or `MauiXamlLoadingSupport`?

Reply:
I think we need the `Maui` prefix for any new MSBuild property, because there might eventually be a WindowsAppSdk property named `EnableXamlLoading`. We can discuss in the meeting today the final name, but I think `MauiEnableXamlLoading` or `MauiXamlLoadingSupport` are both OK.